sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoByteBuddyUtils.java

```diff
@@ -190,18 +191,30 @@
   private static final String DEFAULT_PROTO_GETTER_PREFIX = "get";
   private static final String DEFAULT_PROTO_SETTER_PREFIX = "set";
 
-  static String protoGetterName(String name, FieldType fieldType) {
+  static String protoGetterNameFromSnake(String name, FieldType fieldType) {
     final String camel = CaseFormat.LOWER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, name);
     return DEFAULT_PROTO_GETTER_PREFIX
         + camel
         + PROTO_GETTER_SUFFIX.getOrDefault(fieldType.getTypeName(), "");
   }
 
-  static String protoSetterName(String name, FieldType fieldType) {
+  static String protoSetterNameFromSnake(String name, FieldType fieldType) {
```

Comment:
What do you think about adding a `CaseFormat` parameter to `protoSetterName` instead of having a function for each format?

```suggestion
  static String protoSetterName(String name, FieldType fieldType, CaseFormat caseFormat) {
```